renderer/brave_content_renderer_client.cc

```diff
@@ -45,6 +49,31 @@
 #include "brave/components/playlist/renderer/playlist_render_frame_observer.h"
 #endif
 
+#if BUILDFLAG(ENABLE_WIDEVINE)
+#include "media/base/key_system_info.h"
+#include "third_party/widevine/cdm/widevine_cdm_common.h"
+#endif
+
+namespace {
+void MaybeRemoveWidevineSupport(media::GetSupportedKeySystemsCB cb,
+                                media::KeySystemInfos key_systems) {
+#if BUILDFLAG(ENABLE_WIDEVINE)
```

Comment:
was the existing code for desktop removed? I'm not seeing it in the PR

Reply:
No nothing is removed, I only loosened the condition so it runs both on Android and Desktop now, according to https://github.com/brave/brave-core/pull/18958#discussion_r1273781087.

Reply:
I mean that we wanted it to be removed as discussed in DM. Just commenting for reference.